src/esp/gfx/RenderCamera.h

```diff
@@ -154,7 +180,7 @@ class RenderCamera : public MagnumCamera {
   /**
    * @brief Overload function to render the drawables
    * @param drawables, a drawable group containing all the drawables
-   * @param frustumCulling, whether do frustum culling or not, default: false
```

Comment:
Not sure about the `,` after `drawables`, I suppose it shows as a stray comma in the output?